### DISCUSSION TITLE
用 Taro 3.x 将 formily/react 编译到 wx 小程序，出现样式编译错误。

### DISCUSSION THREAD

**Discussion by rayy-li** [alibaba/formily#1700] [Q&A 2.x] [answered]
实现技术背景：
1、Taro 3.2.13（react、less、ts、默认模板）
（以配置 less-loader 选项：{ javascriptEnabled: true }：https://taro-docs.jd.com/taro/docs/config-detail#minilessloaderoption）
2、formily 2.0.0-beta.72（官方安装流程：已安装 @formily/core @formily/react antd moment @formily/antd 5个包）

执行命令：npm run dev:weapp

微信开发者工具错误：
```
[ WXSS 文件编译错误] ./pages/index/index.wxss
 error at token "*"
  167 |           align-items: center;
  168 | }
> 169 | .ant-formily-editable-content > * {
      |                                ^
  170 |   margin-right: 3rpx;
  171 | }
  172 | .ant-formily-editable-content > *:last-child {
at files://pages/index/index.wxss#169(env: Windows,mp,1.05.2106250; lib: 2.18.0)
```

貌似是不支持 ` > *` 语法编译至wxss？请问有没有一些解决思路？

**Comment by janryWang** [comment, marked as answer]
不要用antd，它本身是给pc用的，标准方案是推荐你自己封装一套小程序端的扩展组件

**Reply by rayy-li** [reply]
嗯嗯是的，不过主要是我们目前自己写 小程序 UI 桥接层 会有些成本过高

**Reply by janryWang** [reply]
成本应该还好吧，移动端也没几个表单组件

**Reply by liuweiGL** [reply]
等 antd-mobile v3 出来可以搞一个，react 在移动端的生态不太好。https://github.com/ant-design/ant-design-mobile/issues/3671#issuecomment-871119428

**Reply by rayy-li** [reply]
> 成本应该还好吧，移动端也没几个表单组件

好的~我预估一下

**Reply by rayy-li** [reply]
> 等 antd-mobile v3 出来可以搞一个，react 在移动端的生态不太好。[ant-design/ant-design-mobile#3671 (comment)](https://github.com/ant-design/ant-design-mobile/issues/3671#issuecomment-871119428)

期待一下~！

**Comment by liuweiGL** [comment]
应该是 less 文件没被 taro 处理？

**Reply by janryWang** [reply]
taro的less语法有限制的吧

**Reply by liuweiGL** [reply]
没有的，走的也是 webpack less-loader

**Reply by janryWang** [reply]
不对，它是把less编译成了wxss，这个是个dsl呀

**Comment by liuweiGL** [comment]
来个 demo 啊！

**Reply by rayy-li** [reply]
demo 就是按照官方taro 和 formily 安装下来，然后配置一下lessloader 让less能执行js，就这样子了~~
